src/devtools/client/debugger/src/actions/breakpoints/index.js

```diff
@@ -198,7 +199,25 @@ export function updateHoveredLineNumber(line) {
     // Set the initial location here as a placeholder to be checked after any async activity.
     dispatch(actions.setHoveredLineNumberLocation(initialLocation));
 
-    await dispatch(setBreakpointPositions({ sourceId: source.id, line }));
```

Comment:
This feels like something where we either want to update `setBreakpointPositions` to always do this for all of the sources that match the URL or add a new `setBreakpointPositionsForURL`.